.ci/updatecli/values.yml

```diff
@@ -0,0 +1,14 @@
+github:
```

Comment:
We can easily copy the existing updatecli pipelines in `.ci/updatecli.d/*.yml` in each APM Agent repository and change the relevant values in this file